go.mod

```diff
@@ -44,3 +44,5 @@ require (
 	gopkg.in/yaml.v2 v2.0.0-20141029210843-d466437aa4ad // indirect
 	k8s.io/client-go v0.0.0-20181219152756-3dd551c0f083 // indirect
 )
+
+replace github.com/docker/libtrust => github.com/containers/libtrust v0.0.0-20190913040956-14b96171aa3b
```

Comment:
Does this really need to be here? I would expect an update to the private branch of containers/image  (testing the code that directly refers to github.com/containers/libtrust), and that transitively brings in github.com/containers/libtrust , which is _actually vendored_ as github.com/containers/libtrust .

This PR does not actually add github.com/containers/libtrust at the expected ordinary path, and it passes pretty much only because it **doesn’t test the c/image PR**.

Reply:
> ```
> go mod edit -replace github.com/containers/image=github.com/lsm5/containers-image@master-openssl-wip
> make vendor
> go get github.com/containers/libtrust
> make vendor
> ```

This seems plausible (maybe we should include the full commands in `CONTRIBUTING.md` as well), but the contents of the PR don’t correspond to these commands AFAICS.

Reply:
Yes, that's what I had in mind as well -> https://github.com/containers/skopeo/pull/720 is passing the vendor checks

Reply:
@lsm5 Drop this override, please.